sycl/test-e2e/syclcompat/math/math_extend_v.cpp

```diff
@@ -0,0 +1,365 @@
+/***************************************************************************
+ *
+ *  Copyright (C) Codeplay Software Ltd.
+ *
+ *  Part of the LLVM Project, under the Apache License v2.0 with LLVM
+ *  Exceptions. See https://llvm.org/LICENSE.txt for license information.
+ *  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCLcompat API
+ *
+ *  math_extend_v.cpp
+ *
+ *  Description:
+ *    math extend-vectorized helpers tests
+ **************************************************************************/
+
+// ===----------- math_extend_vfunc[2/4].cpp ---------- -*- C++ -*
+// --------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===---------------------------------------------------------------------===//
+
+// RUN: %clangxx -fsycl -fsycl-targets=%{sycl_triple} %s -o %t.out
+// RUN: %{run} %t.out
+
+#include <cmath>
+#include <cstdint>
+#include <limits>
+#include <stdio.h>
+#include <sycl/sycl.hpp>
```

Comment:
So, we are not exactly being pushed for it, since `sycl/detail/core.hpp` may be removed at a later date. But if we can use that, it would improve compile times a little bit, which helps reducing testing times in the CI

Reply:
Replaced with `sycl/detail/core.hpp`